onnxruntime/core/graph/model.cc

```diff
@@ -62,11 +63,16 @@ Model::Model(const std::string& graph_name,
     schema_registry->RegisterRegistry(schema_collection);
   }
 
-  auto allow_released_opsets_only =
-      model_load_utils::IsAllowReleasedONNXOpsetsOnlySet();
+  // IsAllowReleasedONNXOpsetsOnlySet() checks for the appropriate env var in the process (i.e.) process-wide
+  // `allow_released_opsets_only` is for this specific Model instance
```

Comment:
Why do we need this model specific instance?
Also, can we rename this to reflect that?

Reply:
By default, model instance creation will not succeed for opset versions > released opset versions. This is a way to programmatically request that model creation be allowed for opsets > released opset versions.

Reply:
`allow_released_opsets_only ` is a paramter passed in via the Model ctor. I was thinking that such a parameter only applied to this Model instance. Do you have a name suggestion ?